app/assets/stylesheets/pages/Login.scss

```diff
@@ -23,9 +23,14 @@
 			text-decoration: underline;
 		}
   }
+
   input {
     border-style: none;
   }
+
+  input.invalid {
+    background-color: $red;
```

Comment:
The red background might be a little aggressive, I'll make some error state mocks for you + help text

Reply:
We'll save this for another PR haha